pkg/commands/cmapflagsandargs.go

```diff
@@ -20,8 +20,8 @@ import (
 	"fmt"
 )
 
-// dataConfig encapsulates the options for add configmap/Secret commands.
-type dataConfig struct {
+// cMapFlagsAndArgs encapsulates the options for add configmap/Secret commands.
```

Comment:
`cMapLagsAndArgs` handles only `ConfigMap`. We can remove Secret from the comments.

Reply:
done.